glib/src/auto/functions.rs

```diff
@@ -534,6 +534,20 @@ pub fn hostname_to_unicode(hostname: &str) -> Option<crate::GString> {
     unsafe { from_glib_full(ffi::g_hostname_to_unicode(hostname.to_glib_none().0)) }
 }
 
+#[cfg(any(feature = "v2_74", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v2_74")))]
+#[doc(alias = "g_idle_add_once")]
+pub fn idle_add_once<P: Fn() + Send + Sync + 'static>(function: P) -> u32 {
```

Comment:
Needs to be implemented manually, see how things are done for `g_idle_add()` in `src/source.rs`

Reply:
slight issue, we already have a manual `idle_add_once` with which is aliasing `#[doc(alias = "g_timeout_add_full")]`

Reply:
You can just mark it `manual`, add the alias and leave it at that